BUILD.gn

```diff
@@ -478,6 +537,15 @@ source_set("electron_lib") {
     ]
   }
 
+  if (enable_builtin_spellchecker) {
+    deps += [
+      ":chrome_spellchecker",
+      "//components/spellcheck/browser",
```

Comment:
`chrome_spellchecker` already depends on `//components/spellcheck`.

Reply:
Yes but some files in this target depend on some spellcheck headers when this buildflag is enabled and `gn check` isn't smart enough to figure it out.  Adding this as a dep seemed cleaner than adding `NOCHECK` comments

Reply:
`public_deps` are there to help for this scenario.